File: app/src/main/java/com/odnovolov/randomuser/MainActivity.java
package com.odnovolov.randomuser;

import android.os.Bundle;

import com.odnovolov.randomuser.utils.BackPressActivity;

public class MainActivity extends BackPressActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/odnovolov/randomuser/MainActivity.kt
package com.odnovolov.randomuser

import android.os.Bundle
import com.odnovolov.randomuser.screen.home.HomeFragment
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class MainActivity : BackPressActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}